src/main/java/ai/datahunters/md/server/analytics/AnalyticsHandler.java

```diff
@@ -0,0 +1,18 @@
+package ai.datahunters.md.server.analytics;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Mono;
+
+import java.util.concurrent.ExecutionException;
+
+@AllArgsConstructor
+@Component
+public class AnalyticsHandler {
+
+    private AnalyticsService service;
+
+    public Mono<AnalyticsData> getAnalytics() throws ExecutionException, InterruptedException {
+            return Mono.just(new AnalyticsData(service.photoCount().get()));
```

Comment:
We are getting really close to the expected solution, types match :tada:, the problem is how you handle the future. Unfortunately we should not block in reactor threads and calling `.get()` on completable future is exactly blocking. There is a utility method on `Mono` which allows building `Mono` from the future, its called `Mono.fromFuture()`